Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor

```diff
@@ -175,7 +175,7 @@
                     var pageNumberString = i.ToString();
                     var pageActive = pageNumber == PaginationContext.CurrentPage;
 
-                    <PaginationItem Display="Blazorise.Display.None.OnMobile.InlineBlock.OnTablet" Disabled="@(pageNumber == PaginationContext.CurrentPage)" Active="@pageActive">
+                    <PaginationItem Display="Blazorise.Display.None.OnMobile.InlineFlex.Row.OnTablet" Disabled="@(pageNumber == PaginationContext.CurrentPage)" Active="@pageActive">

```

Comment:
This is a wide change, is this ok for all providers?

Reply:
Yes, I tried in all providers. Works without visible issues.